humandevprofiles/settings.py

```diff
@@ -0,0 +1,49 @@
+from hurumap.settings import *  # noqa
+
+# insert our overrides before both census and hurumap
+INSTALLED_APPS = ['humandevprofiles'] + INSTALLED_APPS
+
+
+DATABASE_URL = os.environ.get('DATABASE_URL',
+                              'postgresql://humandev:humandev@localhost/humandev')
+DATABASES['default'] = dj_database_url.parse(DATABASE_URL)
+DATABASES['default']['ATOMIC_REQUESTS'] = True
+
+
+HUMANDEV = WAZIMAP
```

Comment:
Let's only use `HURUMAP` for our purposes so that we don't have to overwrite some of the settings from `HURUMAP` that we'd use in templates.